src/main/java/wromaciej/hvac_sim/thermo/matter/fluids/service/FluidData.java

```diff
@@ -10,7 +10,7 @@
  */
 public class FluidData {
 
-    public static void loadLibrary(){
+    public void loadLibrary(){
```

Comment:
hmm, I would do that in some kind constructor at least. Otherwise someone could use `findParameter` method before using `loadLibrary` method